### DISCUSSION TITLE
Planning to write some python example code and documentation for Ory Integration

### DISCUSSION THREAD

**Discussion by initvik** [ory/.github#18] [Ideas]
I am thinking of writing Python Backend service integration examples and documentation for the Ory 

That's the root of the document
https://www.ory.sh/docs/getting-started/

further which can be referenced in the Getting started 
https://console.ory.sh/guides

**Comment by vinckr** [comment]
Hello @initvik 
So sorry about the late answer!

Would be great to have more Python examples! Ideally in this repository: https://github.com/ory/examples. 
Let me know if I can support you with that.
